runtimes/web/src/ui/menu-overlay.ts

```diff
@@ -115,6 +118,7 @@ export class MenuOverlay extends LitElement {
 
     @state() private selectedIdx = 0;
     @state() private netplaySummary: { playerIdx: number, ping: number }[] = [];
+    @state() private vibrateLevel = "";
```

Comment:
I tried just referencing app.vibrateLevel, but wasn't seeing render updates immediately. Is this the correct pattern? Mirror the setting in a local `@state()` variabel?